### ISSUE TITLE
Epic: manual accessibility reviews for vNext RC components

### ISSUE THREAD

**Issue by smhigley** [microsoft/fluentui#21099]
Components that are ready for RC should have a manual a11y review before release:

- [ ] Accordion #21101
- [ ] Avatar #21102
- [ ] Badge (includes CounterBadge and PresenceBadge) #21103
- [ ] Button (includes CompoundButton, MenuButton, SplitButton, and ToggleButton) #20917
- [ ] Card #21104
- [ ] Divider #21105
- [x] Link #19926 
- [ ] Menu #21106
- [ ] Popover #21107
- [ ] Tooltip #21108

**Comment by smhigley** [comment]
Closing in favor of #19779
